narratives/ncov_sit-rep_2020-04-24.md

```diff
@@ -0,0 +1,259 @@
+* [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
+* [Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).
+
+#### External Resources  
+* [Ask a Scientist & FAQs](https://covid19.fas.org/l/en).
+* [WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
+* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
+* [The NYTimes COVID-19 coverage](https://www.nytimes.com/news-event/coronavirus).
+<!-- There is no right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text -->
+# [A note on sampling](https://nextstrain.org/ncov/africa/2020-04-23?d=map&f_region=Africa&p=full)
+We currently have sequences from samples taken in 6 continents, including 7 African countries. This is an incredible achievement -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
+<br><br>
+While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data.
+<br><br>
+For example, the map shows very few sequences from East Africa. This is NOT because COVID-19 isn't circulating in these areas, or that these cases are not as crucial to understand; rather, we just don't have much data available from these areas. The size of each circle on the map indicates how much data is currently available from that area, rather than the true size of the outbreak.
+
+<!-- There is NO right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Many separate introductions have seeded diverse local transmission chains in Africa](https://nextstrain.org/ncov/africa/2020-04-23?d=tree,map&f_region=Africa&p=full&r=country&legend=open)
+  At the time of writing, the [Africa CDC](https://africacdc.org/covid-19/) reports 26,144 confirmed COVID-19 cases and 1,247 deaths across the African Union.
+  We have viral genomes from 83 of these cases from 7 African nations, shown here (in color) against a backdrop of contextual sequences from around the world (gray). We applaud the local public health departments for their rapid response and open data sharing.
+  <br><br>
+  These samples span the full breadth of the tree, indicating that there have been many independent introductions to the African continent.
+  Many of these introductions appear to be from Europe, although this may be skewed by the fact that we have more sequences from Europe than from most other places in the world; we can't rule out the possibility of unsampled, intermediate transmissions elsewhere.
+  <br><br>
+  Not all of these introductions have led to community transmission, but we see clear clusters in some locations.
```

Comment:
```suggestion
  We don't seen evidence that all of these introductions have led to community transmission, but we see clear clusters in some locations.
```
Trying to address @cassiawag 's comment about the fact that sampling will play a role here. We could go into even more detail ("..., as we may not have sampled these clusters, but...") but don't know if we need to.